webapp/app/model/components/__init__.py

```diff
@@ -56,7 +56,7 @@ class DeclarationIncomesProps(StepFormProps):
 
 
 class SelectStmindProps(StepFormProps):
-    pass
+    prev_url: str  # TODO place this at a more general place when implementing more StepFormProps for the LotseFlow
```

Comment:
Why aren't we doing this right away? I think it actually belongs to the `StepFormProps`.